pkg/awssgsetup/aws_security_group_setup.go

```diff
@@ -38,7 +38,7 @@ var DEBUG = 5
 // setupAWSSecurityGroups updates the master and worker security groups used in an
 // OpenShift AWS setup. It sets a time that should be checked before attempting
 // to do another call to setup.
-func SetupAWSSecurityGroups(ctx context.Context, client client.Client) error {
+func SetupAWSSecurityGroups(ctx context.Context, client client.Client, hostedOpenShift bool) error {
```

Comment:
nit, but we can just called this `hosted` I think, given this function is OpenShift only already.

Reply:
Also worth a function-level comment explaining what "hosted" means.